cli/di.container.ts

```diff
@@ -84,7 +84,15 @@ export default function configureContainer(args: any = {}) {
     cache: asFunction((fortaKeystore: string) => flatCache.load('cli-cache', fortaKeystore)).singleton(),
     sleep: asValue((durationMs: number) => new Promise((resolve) => setTimeout(resolve, durationMs))),
 
-    args: asValue(args),
+    args: asFunction(() => {
+      // A lot of our code assumes agentId is present. The cli supports --botId and --agentId but 
```

Comment:
lets just remove support for `--agentId` instead of supporting both here. I think this param only exists for the `info` command